en/docs/setup/deploy-in-openshift.md

```diff
@@ -67,11 +67,11 @@ APK supports OpenShift, which is a Kubernetes distribution with additional featu
 
     1. Identify the `router-service` external IP address.
```

Comment:
need to change here as well

Reply:
Fixed with https://github.com/wso2/docs-apk/pull/323/commits/d35bb770d2262e0952a899cfd5a4dfeedd9bd4c9